depends/packages/native_rust.mk

```diff
@@ -3,16 +3,16 @@ package=native_rust
 # ./contrib/devtools/update-rust-hashes.sh
 # The Rust compiler should use the same LLVM version as the Clang compiler; you
 # can check this with `rustc --version -v`.
-$(package)_version=1.69.0
+$(package)_version=1.70.0
```

Comment:
NACK. Rust 1.70 migrates to LLVM 16; this PR updated Rust without updating Clang.